src/client.js

```diff
@@ -3020,6 +3020,25 @@ module.exports.CRYPTO_ENABLED = CRYPTO_ENABLED;
  */
 
 /**
+ * Fires whenever an error occurs when call.js sendEvent encounters an error.
+ * <p>
+ * This is required to allow UnknownDeviceError to propagate from client.sendEvent, whilst
```

Comment:
it's not specific to unknowndeviceerror though? it's about all event-transmission errors?